### PR TITLE
MVC View example

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi team,

I have updated the MVC Views examples for v8. Let me know if it needs any more additions.

Refers issue [1877](https://github.com/umbraco/UmbracoDocs/issues/1877)

Poornima

### CODE REVIEW

File: Reference/Templating/Mvc/examples.md [hunk 1]
```diff
@@ -1,47 +1,28 @@
...
-
-```csharp
-@Umbraco.Field("bodyContent", insertBefore : "<h2>", insertAfter : "</h2>")
-```
-
 ## Rendering the raw value of a field from IPublishedContent
 
 ```csharp
-@Model.Content.Properties["bodyContent"].Value
+@Model.Properties["bodyContent"].Value
```
Comment (sofietoft) [inline Reference/Templating/Mvc/examples.md]: I've tested this, and can't seem to get it working 😅
Might very well be doing something wrong .. In which context did you get an output using this?

File: Reference/Templating/Mvc/examples.md [hunk 2]
```diff
@@ -1,47 +1,28 @@
...
 ```
 
 Or alternatively:
```
Comment (sofietoft) [inline Reference/Templating/Mvc/examples.md]: Think `@Model.Value("")` should be the *standard* as opposed to "the alternative one" 🤔 

What do you think?

File: Reference/Templating/Mvc/examples-v7.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+---
+versionFrom: 7.0.0
+needsV8Update: "true"
```
Comment (sofietoft) [inline Reference/Templating/Mvc/examples-v7.md]: `needsV8Update` can be removed here, once this PR is merged, right? 😄

File: Reference/Templating/Mvc/examples.md (hunk 1 above)
Comment (poornimanayar) [reply Reference/Templating/Mvc/examples.md]: Facepalm! Let me fix them!

Comment (poornimanayar) [reply Reference/Templating/Mvc/examples.md]: This should now be fixed. :-)

**Comment by sofietoft** [comment]
Thanks for making the small changes @poornimanayar ! 😄 

Looks like we're good here. I'll merge 🎉

**Pull request merged**